Terminal.Gui/Views/TableView/TableStyle.cs

```diff
@@ -93,24 +93,39 @@ public class TableStyle {
 	/// </summary>
 	public bool ShowHorizontalBottomline { get; set; } = false;
 
+	/// <summary>
+	/// True to invert the colors of the entire selected cell in the <see cref="TableView"/>.
+	/// Helpful for when <see cref="TableView.FullRowSelect"/> is on, especially when the <see cref="ConsoleDriver"/> doesn't show
+	/// the cursor
+	/// </summary>
+	public bool InvertSelectedCell { get; set; } = false;
+
 	/// <summary>
 	/// True to invert the colors of the first symbol of the selected cell in the <see cref="TableView"/>.
 	/// This gives the appearance of a cursor for when the <see cref="ConsoleDriver"/> doesn't otherwise show
 	/// this
 	/// </summary>
 	public bool InvertSelectedCellFirstCharacter { get; set; } = false;
 
-	// TODO: Fix this, or just remove it [and SeparatorSymbol]?
+	// NOTE: This is equivalent to True by default after change to LineCanvas borders and can't be turned off
+	// without disabling ShowVerticalCellLines, however  SeparatorSymbol and HeaderSeparatorSymbol could be
+	// used to approximate the previous default behavior with FullRowSelect
+	// TODO: Explore ways of changing this without a workaround
 	/// <summary>
 	/// Gets or sets a flag indicating whether to force <see cref="ColorScheme.Normal"/> use when rendering
 	/// vertical cell lines (even when <see cref="TableView.FullRowSelect"/> is on).
 	/// </summary>
 	//public bool AlwaysUseNormalColorForVerticalCellLines { get; set; } = false;
 
 	/// <summary>
-	/// The symbol to add after each cell value and header value to visually seperate values (if not using vertical gridlines)
+	/// The symbol to add after each header value to visually seperate values (if not using vertical gridlines)
+	/// </summary>
+	public char HeaderSeparatorSymbol { get; set; } = ' ';
```

Comment:
Per our guidelines and just because it's a plain ol' good idea, how about defining constants for these instead of the char literal?

Fast way to do that would be to highlight the `' '` in one spot, press ctrl-alt-c, to invoke the resharper introduce constant refactoring, choose the option that says "replace x instances", which will cover any it thinks are relevant within the same scope, and then pick a name for the constant such as `SingleSpace` (or anything else that would make sense and also not be a potential conflict with any future strings or other types with a similar value/purpose).

And of course not necessarily in this PR. Just making note of a place for us to dogfood our guidelines.

Reply:
I think that if this is to be a constant it would make more sense as `DefaultHeaderSeparatorSymbol` rather than a global find and replace.

I think that approach would also work better with ConfigurationManager which has some precedent for changing defaults via settings files